packages/stacked_tools/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.4-beta.0
```

Comment:
For beta we keep the version the same and use beta.1 instead. So next version stays 0.2.3, we only update the beta.X number

Reply:
I changed the patch version because it adds a new configuration option, but probably could be part of the previous version. What leaves me with doubts is we are also changing the default view template.